test/units/default_setup.js

```diff
@@ -0,0 +1,16 @@
+QUnit.module("Default setup ( #default_settings )", {
+  beforeEach: function(){
+    $('#default_settings .additional_information').showWhenThis( {
+      change_selector: $('#default_settings [name="show_additional_information"]')
```

Comment:
Line is too long.<br>Identifier 'change_selector' is not in camel case.<br>Mixed double and single quotes.